src/planning/src/Logger.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
```

Comment:
Should port the code to not require this header (and remove it).

Reply:
Yes, also needs to be moved to a util folder